meta/labs/README.md

```diff
@@ -3,6 +3,24 @@
 This document will contain a brief overview of our labs program and what it'll
 entail going into the 2015 fall semester.
 
+## Format
+
+The Labs program will follow this format:
+
+1. Workshops
+   - These are based on the workshops from Hack Camp
```

Comment:
I would include what the objective of what Labs is and leave it at that. Perhaps link to the workshops as something that clubs can use but not that they are mandatory.